lib/tessel/access-point.js

```diff
@@ -146,12 +151,16 @@ Tessel.prototype.createAccessPoint = function(opts) {
 
   return this.simpleExec(commands.getAccessPointSSID())
     .then(() => {
+      log.spinner.stop();
+
       // When an AP exists, change the status to
       // reflect an "update" vs. "create":
       status = 'Updated Access Point successfully. ';
       return setupAccessPoint();
     })
     .catch(() => {
+      log.spinner.stop();
```

Comment:
`setupAccessPoint` calls this after the other async actions of `setAccessPoint` and `setLanNetwork`. Could this be removed so the spinner continues until the access point is actually finished being setup?